mi/dataset/parser/flort_dj_dcl.py

```diff
@@ -135,6 +156,62 @@
     ('raw_internal_temp',           SENSOR_GROUP_INTERNAL_TEMPERATURE,  int)
 ]
 
+# Combined CTDBP_FLORT Sensor data record:
+#
+# match a single line from a platform generated by a combined CTDBP_CDEF_DCL
+# with a FLORT_D plugged into it and process the FLORT data only.
+#
+#   Timestamp <optional>LOGGER_ID <optional># SensorData Date Time
+#   where SensorData are comma-separated floats and unsigned integer numbers
+#         Date is DD MMM YYYY format
+#         Time is HH:MM:SS format
+LOGGER_ID = START_METADATA + ANY_CHARS_REGEX + END_METADATA
+LOGGER_ID += COLON                          # [ id_string ]:
+
+CTDBP_FLORT_PATTERN = TIMESTAMP             # dcl controller timestamp
```

Comment:
These regex constructs are truly horrible. They completely obfuscate the actual regular expression and groupings which are necessary to extract the data as well as make the pattern match overly explicit which can lead to future parsing errors. I recognize that it was the practice of the previous author of the code so no change is necessary, but it should be noted that this is not a good way to parse data.